test/runners/python_spec.js

```diff
@@ -329,9 +329,12 @@ describe('Fixed and new features', function() {
         languageVersion: v,
         testFramework: 'cw-2',
         code: 'a = 1',
-        fixture: 'print(1, "2", u"\\uac00")',
+        fixture: 'test.uni_print(1, "a", u"\\uac00", [314159, "b", u"\\uac01"])',
```

Comment:
For Python 2, this is printing:
```
1 a &#44032; [314159, 'b', u'&#44033;']
```
![image](https://user-images.githubusercontent.com/639336/36136498-a945a0f4-1045-11e8-8480-27c1fa327bf1.png)

https://travis-ci.org/Codewars/codewars-runner-cli/jobs/340822289#L1401

Reply:
Should I remove the `u` flag in front of the literal? I think I'll leave it there, because `str` and `unicode` are two different things in Py2.

Reply:
Ok, I didn't realize that was intentional. So it'll be rendered like this:
![image](https://user-images.githubusercontent.com/639336/36137439-1bdda37e-104a-11e8-8f91-d5cd1eac03bf.png)

Reply:
Yes, I tested the output using my kumite on CW, but I don't know how it'll be displayed on Qualified.